applications/FluidDynamicsApplication/python_scripts/trilinos_navier_stokes_solver_fractionalstep.py

```diff
@@ -90,13 +75,12 @@ def __init__(self, model, custom_settings):
         self.min_buffer_size = 3
         self.element_has_nodal_properties = True
 
-        ## Construct the linear solvers
-        self.pressure_linear_solver = trilinos_linear_solver_factory.ConstructSolver(self.settings["pressure_linear_solver_settings"])
-        self.velocity_linear_solver = trilinos_linear_solver_factory.ConstructSolver(self.settings["velocity_linear_solver_settings"])
-
         self.compute_reactions = self.settings["compute_reactions"].GetBool()
 
-        KratosMultiphysics.Logger.PrintInfo("TrilinosNavierStokesSolverFractionalStep","Construction of TrilinosNavierStokesSolverFractionalStep solver finished.")
+        self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.OSS_SWITCH, self.settings["oss_switch"].GetInt())
+        self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.DYNAMIC_TAU, self.settings["dynamic_tau"].GetDouble())
```

Comment:
shouldn't those things also be done in the baseclass-constructor?

EDIT: I checked the implementation, it seems like some minor refactoring needs to be done, bcs right now the Trilinos-solver calls the parent of the parent :sweat_smile: 

but please in a future PR ;)

Reply:
Agree.This was a workaround we did in the past (I guess because the defaults did not completely match). I already added a note in the Fluid Dynamics Application Project.